### ISSUE TITLE
MADP configuration notes

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2337]
@urapadmin: I'm catching some slight oddities for you to look at and will record them here.

- [ ] MADP is cut off in the upper right
- [ ] tag analysis in registration now says L-O-C-U-S analysis by tag and the headings are also weird when one enters it

**Comment by urapadmin** [comment]
the first is me waiting for the final logo
the second is a configuration key somewhere

**Comment by lbestock** [comment]
- [ ] configuration: "cache only direct image links" has wound up on the identification tab

**Comment by lbestock** [comment]
There are some really weird things floating around in configuration. Since when and why and who made term_for_ceramic_fragment into "fragy"?? And there are several of these L-O-C-U-S or U-N-I-T thingees now.

**Comment by urapadmin** [comment]
> There are some really weird things floating around in configuration. Since when and why and who made term_for_ceramic_fragment into "fragy"?? And there are several of these L-O-C-U-S or U-N-I-T thingees now.

you seem to have gotten rid of those. Can't find them anymore. The reason why I name these things like that is so that they would stick out and remind you that they need configuration. "fragy", though, I don't know where that came from.

**Comment by urapadmin** [comment]
I note here that the current configuration is not adding the unit as a prefix to the locus. So when I add a locus to unit PLG-1 I get a "950" and that's it. Is that what you wanted?

**Comment by luizaogs** [comment]
Hmm sorry Laurel, I don't think I specified this. Having the unit as a prefix might be good just because I think each chapel will be a unit, so it would be helpful to have A-950 or what have you. If it's not too much trouble.

**Comment by urapadmin** [comment]
tis'nt

**Comment by urapadmin** [comment]
sorry: 'tis'nt

**Comment by urapadmin** [comment]
butchering my Shakespeare with wrong punctuation

**Comment by luizaogs** [comment]
Alack!

**Comment by urapadmin** [comment]
I added the domain as a prefix to the partitioned locus number. Just be aware that the locus is being created according to the partion only, so when creating a locus in PLG-1 I get a PLG-1-950 (first number in my partition). When I then create a locus in a unit A of your real site (let's call the site MA here) I get a MA-1-951. The number comes from the system-wide partition.

**Comment by urapadmin** [comment]
Also note that currently samples, small find an everything is running on the same sequence. So I add a sample: It gets PLG-1-1. I add a bulk it gets PLG-1-2 etc. So the last part is independent of the collected material type. It's what we usually do but it does not have to be that way.

**Comment by luizaogs** [comment]
Sorry, so the prefix is the site and not the unit then? Slightly confused.

The cm numbering is good (I doubt we will have cm...)

**Comment by urapadmin** [comment]
Sorry, that was just me making a unit PLG-1. Confusing indeed. But now you can look at it live and that might disperse the confusion. 
If that's not what you had in mind for locus numbering - we can change that now to (almost) whatever you want. 'tis rather versatile.

**Comment by urapadmin** [comment]
erm, by the way, the header and logo in the madp current recording app is not meant seriously. I just had to put in something so I made it a bit crazy. When Maarten comes along with a better logo, it will get less crazy. I think.

**Comment by luizaogs** [comment]
Well now I have to see what that is...

**Comment by urapadmin** [comment]
it's pinkish. Nothing I would usually go for :o)

**Comment by luizaogs** [comment]
Ha. And word art!

**Comment by urapadmin** [comment]
I first tried Comic Sans so that everybody's eyes would hurt but I was too feeblish. Could not make myself look at it for a second.

**Comment by luizaogs** [comment]
I'm so sick of designing a powerpoint presentation currently that I'm very tempted to just make everything Comic Sans and be done with it. Maybe in different colors.

**Comment by urapadmin** [comment]
that will make half of your audience faint. (Or I should hope so)
By the way: For my friendship book (if that is a term let alone a thing here): Do you have a favourite colour and a most hated colour?

**Comment by luizaogs** [comment]
That would be great! I wouldn't need to present.

I had never heard of friendship books but I have now looked them up and they sound (and look) lovely :) Favorite color is most definitely blue. I don't think I really have a most hated color ... but if I had to pick maybe bright yellow.

**Comment by urapadmin** [comment]
Should you need a break from your presentation I would like to see "blue" specified in RGB, CMYK. Or if you fancy HSL, that' fine too :o) Then I can play around with the somewhat addictive https://coolors.co

**Comment by luizaogs** [comment]
Ha that was harder than I expected!

<img width="657" alt="Screenshot 2023-11-06 at 6 04 29 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/4a8b0b26-c159-4d23-ae0a-ff5186ac6ca7">

**Comment by luizaogs** [comment]
And I just checked the identifier creation for loci and it looks good to go.

**Comment by urapadmin** [comment]
ha. Just what I need while my tomato sauce is simmering. Driest of Martinis, Coolors and out comes a MADP Kiosk color scheme...

**Comment by luizaogs** [comment]
Ha ha. I look forward to seeing it.

**Comment by urapadmin** [comment]
> One configuration note from me: we currently have volume and weight overlapping in the sample form so I can't read them. I guess it's the issue in #2285.

ja. You have to decide between either weight or volume per collected material type. I was under the assumption that it is an either or and since I ran out of space on that layout I just piled them on top of each other. 

Or do you need both at the same time for one of the cm types?

**Comment by luizaogs** [comment]
No, it should be weight for both bulk and small find and volume for sample. Bulk and small find look ok already so I guess it's just a matter of turning off weight for sample.

**Comment by urapadmin** [comment]
ah, now I see. That is a bug. it is only the labels that are both visible all of a sudden, the input fields are fine. So configuration is okay, it needs a bug fix.

**Comment by luizaogs** [comment]
Oh yes, sure. The standard ones would be good.

**Comment by urapadmin** [comment]
> Oh yes, sure. The standard ones would be good.

done

**Comment by luizaogs** [comment]
Thank you.

**Comment by urapadmin** [comment]
so, if you give it the go tomorrow, I'll shut down the internet server and transfer everything local and then we can send the computer to you on Friday.

**Comment by luizaogs** [comment]
I believe it's ready to go on my end.

**Comment by luizaogs** [comment]
And closing this...
